js/server/tests/resilience/resilience-synchronous-repl-cluster.js

```diff
@@ -321,6 +322,26 @@ function SynchronousReplicationSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     setUp : function () {
+      if (!inquiry_checked) {
```

Comment:
This is kind of uncommon.
Why are the inquiry tests happening in `setUp` and not in their own test function(s)?